pyproject.toml

```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/hiboxsystems/marge-bot"
 keywords = ["merge request", "gitlab"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
This means "3.8 or greater, but less than 4.0", right?

Reply:
No idea :grin:

Reply:
Seems about right.

https://python-poetry.org/docs/dependency-specification/

![image](https://github.com/hiboxsystems/marge-bot/assets/1417619/b6d37ba4-13c5-428b-8250-f069d11228c9)

Reply:
OK, kudos for confirming.